sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -1048,31 +1031,155 @@ def hpat_pandas_stringmethods_isdecimal_impl(self):
+
+    def hpat_pandas_stringmethods_title_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
+            result[idx] = self._data._data[idx].title()
+
+        return pandas.Series(result, self._data._index, name=self._data._name)
+
+    return hpat_pandas_stringmethods_title_impl
+
+
+@sdc_overload_method(StringMethodsType, 'swapcase')
+def hpat_pandas_stringmethods_swapcase(self):
+    ty_checker = TypeChecker('Method swapcase().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_swapcase_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
+            result[idx] = self._data._data[idx].swapcase()
+
+        return pandas.Series(result, self._data._index, name=self._data._name)
+
+    return hpat_pandas_stringmethods_swapcase_impl
+
+
+@sdc_overload_method(StringMethodsType, 'casefold')
+def hpat_pandas_stringmethods_casefold(self):
+    ty_checker = TypeChecker('Method casefold().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_casefold_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
+            result[idx] = self._data._data[idx].casefold()
+
+        return pandas.Series(result, self._data._index, name=self._data._name)
+
+    return hpat_pandas_stringmethods_casefold_impl
+
+
+see_also_for_ismethods = '''
+                        :ref:`Series.str.isalpha <pandas.Series.str.isalpha>`
+                            Check whether all characters are alphabetic.
+                        :ref:`Series.str.isnumeric <pandas.Series.str.isnumeric>`
+                            Check whether all characters are numeric.
+                        :ref:`Series.str.isalnum <pandas.Series.str.isalnum>`
+                            Check whether all characters are alphanumeric.
+                        :ref:`Series.str.isdigit <pandas.Series.str.isdigit>`
+                            Check whether all characters are digits.
+                        :ref:`Series.str.isdecimal <pandas.Series.str.isdecimal>`
+                            Check whether all characters are decimal.
+                        :ref:`Series.str.isspace <pandas.Series.str.isspace>`
+                            Check whether all characters are whitespace.
+                        :ref:`Series.str.islower <pandas.Series.str.islower>`
+                            Check whether all characters are lowercase.
+                        :ref:`Series.str.isupper <pandas.Series.str.isupper>`
+                            Check whether all characters are uppercase.
+                        :ref:`Series.str.istitle <pandas.Series.str.istitle>`
+                            Check whether all characters are titlecase.
+                        '''
+
+
+see_also_for_conv_methods = '''
```

Comment:
What do you mean by "conv"?

Reply:
the description of almost all of these methods begins with the word "converts". Based on this, I decided to unite in this way.

Reply:
"Conv" is a common contraction for convolution layers in DL, that's why this name is a bit confusing